src/main/java/javax/security/enterprise/authentication/mechanism/http/LoginToContinue.java

```diff
@@ -51,7 +51,7 @@
 import javax.resource.spi.AuthenticationMechanism;
 
 /**
- * The <code>LoginToContinue</code> annotation provides an application the ability to declaratively 
+ * The <code>LoginToContinue</code> annotation provides an application the ability to declarative
```

Comment:
I was always under the impression that declaratively is not a correct English word (but I'm not native English so I'll remove this change)